File: CPTM.ILA.Web/DTOs/UserDto.cs
using System.Collections.Generic;
using CPTM.ILA.Web.Models.AccessControl;

namespace CPTM.ILA.Web.DTOs
{
    public class UserDto
    {
        public int Id { get; set; }
        public string Username { get; set; }
        public bool IsComite { get; set; }
        public bool IsDPO { get; set; }
        public bool IsSystem { get; set; }
        public Group OriginGroup { get; set; }
        public ICollection<Group> Groups { get; set; }
        public string Nome { get; set; }
    }
}

File: source/frontend/CPTM.ILA.Frontend/src/shared/models/DTOs/user-dto.ts
import { Group } from "../access-control/group.model";

export interface UserDto {
    id: number;
    username: string;
    isComite: boolean;
    isDPO: boolean;
    isSystem: boolean;
    originGroup: Group;
    groups: Group[];
    groupAccessExpirationDate: Date;
    nome: string;
}
